internal/filer_test.go

```diff
@@ -736,6 +736,44 @@ func TestAccWorkspaceFilesExtensionsDirectoriesAreNotNotebooks(t *testing.T) {
 	assert.ErrorIs(t, err, fs.ErrNotExist)
 }
 
+func TestAccWorkspaceFilesExtensionsNotebooksAreNotReadAsFiles(t *testing.T) {
+	t.Parallel()
+
+	ctx := context.Background()
+	wf, _ := setupWsfsExtensionsFiler(t)
+
+	// Create a notebook
+	err := wf.Write(ctx, "foo.ipynb", strings.NewReader(jupyterNotebookContent1))
+	require.NoError(t, err)
```

Comment:
This will conflict with #1827 which removes this fixture. Make sure you sequence these PRs.

Reply:
Fixed